### PR TITLE
[bitnami/postgresql] Allow specifying different resource requests/limits on slaves

### PR DESCRIPTION
**Pull request by vaghetti**
**Description of the change**
Adds the ability to specify a different resource request for the slave replicas

**Benefits**
If your use case uses the slave replicas as read only replicas, but not intending to use them as a replacement for the master replica in case it goes down, it can make sense to allocate less resources to it. With the current behavior of the helm chart, the resource requirements for master and slave must always be the same.

**Possible drawbacks**
None that I can think of. The change is backwards compatible, and can be safely ignored by people who don't need it. 

**Applicable issues**
Couldn't find any

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

**Comment by vaghetti** [comment]
Seems like I messed up something when forking the repo. I'll fix it and re-open the PR.

File: bitnami/postgresql/README.md [hunk 1]
```diff
@@ -182,6 +182,7 @@ The following tables lists the configurable parameters of the PostgreSQL chart a
...
 | `slave.anotations`                            | Map of annotations to add to the statefulsets (postgresql slave)                                                                                                                                                                                                                                                                                                                                                                                                           | `{}`                                                          |
+| `slave.resources`                             | CPU/Memory resource requests/limits override for slaves. Will fallback to `values.resources` of not defined.                                                                                                                                                                                                                                                                                                                                                               | `{}`                                                          |
```
Comment (juan131) [inline bitnami/postgresql/README.md:185]: Typo: `~of~ not defined` -> `if not defined`

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks so much for this PR! 

Could you please fix the small typo? Once done, we can go ahead with the merging process.

File: bitnami/postgresql/README.md (hunk 1 above)
Comment (vaghetti) [reply bitnami/postgresql/README.md]: Oops, typo fixed.

**Review by juan131 — APPROVED** [review]
LGTM

**Pull request merged**